src/Domain/CoordinateMaps/TimeDependent/Translation.cpp

```diff
@@ -348,7 +348,7 @@ std::array<tt::remove_cvref_wrap_t<T>, Dim> Translation<Dim>::piecewise_helper(
   for (size_t i = 0; i < Dim; i++) {
     gsl::at(result, i) = gsl::at(source_coords, i);
   }
-  auto radius = magnitude(result);
+  const tt::remove_cvref_wrap_t<T> radius = magnitude(result);
```

Comment:
This was giving a Blaze expression referencing `result`, so reading from it took into account changes to `result` that had happened after here until that point.

I believe all the other lines were correct, but some of them are probably more efficient with these changes, so that's also good.